devices/immax.js

```diff
@@ -163,4 +163,53 @@ module.exports = [
             // eslint-disable-next-line
             exposes.enum('keep_time', ea.STATE_SET, ['0', '30', '60', '120', '240']).withDescription('PIR keep time in seconds')],
     },
+    {
+        fingerprint: [{modelID: 'TS0601', manufacturerName: '_TZE200_4eeyebrt'}],
```

Comment:
This can be merged with https://github.com/Koenkk/zigbee-herdsman-converters/blob/87c2535737b04e6d69703ad77dd0281e4cd0d4f8/devices/tuya.js#L1139